### PR TITLE
Math Magicians components and ui

### PR DESCRIPTION
**Pull request by gracekabaghe**
- Created a directory called components.
- Inside components, created a new Calculator.js file.
- In Calculator.js, created a React component that matches this design:

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,30 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
 
 function App() {
```
Comment (rloterh) [inline src/App.js]: - [x] Kindly ensure that the _main_(App) component is a **stateful/ class-based** component, as specified.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import '../index.css';
+
+const Calculator = () => (
```
Comment (rloterh) [inline src/components/Calculator.js]: - [x] Kindly ensure that the _Calulator_ component is a **stateful/ class-based** component, as specified.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Grace, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- No linter errors

## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  const count = useState(0);
+
+  return (
+    <div className="calculator-flex">
+      <div className="output">{count}</div>
+      <div className="buttons">
+        <button type="button" className="button">AC</button>
+        <button type="button" className="button">+/-</button>
+        <button type="button" className="button">%</button>
+        <button type="button" className="button highlight">&divide;</button>
```
Comment (dasileker) [inline src/components/Calculator.js]: - [ ] As my colleague said the Calculator.js should be a class-based app.

mmm

File: src/App.js [hunk 4]
```diff
@@ -1,30 +1,8 @@
...
-  return (
-    <div className="App">
```
Comment (dasileker) [inline src/App.js]: - [ ] the same goes for the App.js file.


ooo

**Review by dasileker — CHANGES_REQUESTED** [review]
## **Hi @gracekabaghe,** 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ArthurGC — APPROVED** [review]
# Approved project!  💯 ✅ 🥳 🎈 🎉
Hi  @gracekabaghe 

Amazing job implementing suggestions from the previous reviewer! 💯 
Your project is complete! :shipit: 
Congratulations! 🎉 



## Aspects to improve ♻️
- [OPTIONAL] It would be better to add a customized README instead of README that comes from React by default. A good start would be the [MICROVERSE README](https://github.com/microverseinc/readme-template)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**